packages/composable-controller/src/ComposableController.test.ts

```diff
@@ -97,81 +77,25 @@ class BarController extends BaseController<never, BarControllerState> {
   }
 }
 
-const setupControllers = () => {
-  const mainMessenger = new ControllerMessenger<
-    never,
-    NetworkControllerStateChangeEvent
-  >();
-
-  const messenger: NetworkControllerMessenger = mainMessenger.getRestricted({
-    name: 'NetworkController',
-    allowedEvents: ['NetworkController:stateChange'],
-    allowedActions: [],
-  });
-
-  const composableMessenger: ComposableControllerRestrictedMessenger =
-    mainMessenger.getRestricted({
-      name: 'ComposableController',
-      allowedEvents: ['NetworkController:stateChange'],
-      allowedActions: [],
-    });
-
-  const networkController = new NetworkController({
-    messenger,
-    infuraProjectId: 'potato',
-  });
-  const preferencesController = new PreferencesController();
-
-  const assetContractController = new AssetsContractController({
-    onPreferencesStateChange: (listener) =>
-      preferencesController.subscribe(listener),
-    onNetworkStateChange: (listener) =>
-      messenger.subscribe('NetworkController:stateChange', listener),
-  });
+interface BazControllerState extends BaseState {
+  baz: string;
+}
+class BazController extends BaseController<never, BazControllerState> {
+  defaultState = {
+    baz: 'baz',
+  };
 
-  const nftController = new NftController({
-    onPreferencesStateChange: (listener) =>
-      preferencesController.subscribe(listener),
-    onNetworkStateChange: (listener) =>
-      messenger.subscribe('NetworkController:stateChange', listener),
-    getERC721AssetName: assetContractController.getERC721AssetName.bind(
-      assetContractController,
-    ),
-    getERC721AssetSymbol: assetContractController.getERC721AssetSymbol.bind(
-      assetContractController,
-    ),
-    getERC721TokenURI: assetContractController.getERC721TokenURI.bind(
-      assetContractController,
-    ),
-    getERC721OwnerOf: assetContractController.getERC721OwnerOf.bind(
-      assetContractController,
-    ),
-    getERC1155BalanceOf: assetContractController.getERC1155BalanceOf.bind(
-      assetContractController,
-    ),
-    getERC1155TokenURI: assetContractController.getERC1155TokenURI.bind(
-      assetContractController,
-    ),
-    onNftAdded: jest.fn(),
-  });
+  override name = 'BazController';
 
-  const tokensController = new TokensController({
-    onPreferencesStateChange: (listener) =>
-      preferencesController.subscribe(listener),
-    onNetworkStateChange: (listener) =>
-      messenger.subscribe('NetworkController:stateChange', listener),
-  });
+  constructor() {
+    super();
+    this.initialize();
+  }
 
-  return {
-    messenger,
-    composableMessenger,
-    networkController,
-    preferencesController,
-    assetContractController,
-    nftController,
-    tokensController,
-  };
-};
+  updateBaz(baz: string) {
+    super.update({ baz });
+  }
```

Comment:
I see you're copying the pattern from `BarController` but is this method actually used/needed?

Reply:
Nope, it's unused. Removed here: e64c024d3d31dd13702664f20389456664953bda